src/meta/topics/building-from-src.md

```diff
@@ -0,0 +1,198 @@
+# Building from src
+
+## Unix / Macintosh
+
+Prerequisites:
+
+* `gcc` and `g++` 4.8 or newer, or
+* `clang` and `clang++` 3.4 or newer
+* Python 2.6 or 2.7
+* GNU Make 3.81 or newer
+* libexecinfo (FreeBSD and OpenBSD only)
+
+```text
+$ ./configure
+$ make
+$ [sudo] make install
+```
+
+If your Python binary is in a non-standard location or has a
+non-standard name, run the following instead:
+
+```text
+$ export PYTHON=/path/to/python
+$ $PYTHON ./configure
+$ make
+$ [sudo] make install
+```
```

Comment:
This is all copy/pasted from the io.js readme.md, so if it's wrong here it's also wrong there.

It looks like _you_ wrote that here: https://github.com/nodejs/io.js/commit/13d86a86d615b3ca32a0870b2b4921e0e611309d. Has something changed since then which would make that no longer valid?